projects/schematics/src/add-spartacus/schema.json

```diff
@@ -34,6 +34,16 @@
       "type": "boolean",
       "description": "Overwrite whole app.component.html file.",
       "default": false
+    },
+    "pwa": {
+      "type": "boolean",
+      "description": "Execute additional command for PWA configuration.",
```

Comment:
Maybe just "Add PWA support to the project", more description of an actual benefit than technical details.